model/reactions.tsv

```diff
@@ -7191,7 +7191,6 @@ rxns	rxnKEGGID	rxnBiGGID	rxnEHMNID	rxnHepatoNET1ID	rxnREACTOMEID	rxnRecon3DID	rx
 "MAR08412"	""	"O2Stn"	""	""	""	"O2Stn"	"MNXR102089"	"HMR_8412"	"RCR20749"	""	0	""	""	"HMR_8412"
 "MAR08447"	""	"CYTK1"	""	""	""	"CYTDtn"	"MNXR97047"	"HMR_8447"	"RCR20750"	""	0	""	""	"HMR_8447"
 "MAR08476"	""	""	""	""	""	"HMR_8476"	""	"HMR_8476"	"RCR21086"	"2.A.57.1.4"	0	""	""	"HMR_8476"
-"MAR08639"	""	""	""	""	""	"HMR_8639"	""	"HMR_8639"	"RCR21087"	""	0	""	""	"HMR_8639"
```

Comment:
Could you please add this deleted reaction (`MAR08639`) to the `deprecatedReactions.tsv` file (in `data/deprecatedIdentifiers/`)? You can just copy the deleted row from `reactions.tsv` and append it to the end of `deprecatedReactions.tsv`.

Reply:
I think it is ok now